content/docs/latest/languagemanual-commands_34838882.md

```diff
@@ -25,6 +25,7 @@ Commands are non-SQL statements such as setting a property or adding a resource.
 | <query string> | Executes a Hive query and prints results to standard output. |
 | source FILE <filepath> | Executes a script file inside the CLI. |
 | compile `<groovy string>` AS GROOVY NAMED <name> | This allows inline Groovy code to be compiled and be used as a UDF (as of Hive [0.13.0](https://issues.apache.org/jira/browse/HIVE-5252)). For a usage example, see [Nov. 2013 Hive Contributors Meetup Presentations – Using Dynamic Compilation with Hive](https://cwiki.apache.org/confluence/download/attachments/27362054/HiveContrib-Nov13-groovy_plus_hive.pptx?version=1&modificationDate=1385171856000&api=v2). |
+| show processlist                                                 | Displays the operations running on Hiveserver2 and provides session and operation details. It will help troubleshoot issues with the Hiveserver2 like identify inappropriate connections to terminate them, identify long running hive queries                                                                                                                                                                                                                                                                                                                                                                                                                         |
```

Comment:
* HiveServer2 vs Hiveserver2: the first form appears more often than the second.
* "It will help" vs "It helps": The future tense is not necessary since the feature is implemented and part of the product.
* Since we don't have versioned documentation it is useful to include the JIRA id where users can find in which Hive version the feature is present.

Minor reformulation based on the above:
"Displays information about the operations currently running on HiveServer2. It helps to troubleshoot issues such as long running queries, connection starvation, etc. The command was introduced in [HIVE-27829](...)."

Reply:
Updated text as recommended.